vNext/test/TestInternalDtosRefOrleans/TestInternalDtosRefOrleans.csproj

```diff
@@ -0,0 +1,55 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Begin Orleans: Without these lines the project won't build properly -->
+  <!-- Set path to ClientGenerator.exe -->
+  <!--End Orleans -->
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup Label="Configuration">
+    <DefineConstants>NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
+    <RootNamespace>UnitTests.DtosRefOrleans</RootNamespace>
+    <AssemblyName>TestInternalDtosRefOrleans</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RuntimeIdentifier>win</RuntimeIdentifier>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\..\..\test\TestInternalDtosRefOrleans\**\*.cs" Exclude="      ..\..\..\test\TestInternalDtosRefOrleans\obj\**\*.cs;       ..\..\..\test\TestInternalDtosRefOrleans\bin\**\*.cs;       ..\..\..\test\TestInternalDtosRefOrleans\Properties\*.cs" />
```

Comment:
The Excludes here lose readability because of the auto-formatting :( maybe the whitespace should just be removed altogether